File: Java Interview Questions/Bubble.java
class Bubble{

    public static void BubbleSort(int a[]){
        int t;
        int n=a.length;
        for(int i=0; i<n-1; i++){
            for(int j=0; j<n-1-i;j++){
                if(a[j]>a[j+1]){
                    t=a[j];
                    a[j]=a[j+1];
                    a[j+1]=t;
                }
            }
        }
    }
    static void printArray(int a[]) 
    { 
        int n = a.length; 
        for (int i = 0; i < n; ++i) 
            System.out.print(a[i] + " "); 
        System.out.println(); 
    } 
    public static void main(String[] args) {

          int arr[]={3,5,7,9,2,4,8};
          BubbleSort(arr);
            printArray(arr);
    }
}

File: Sorting/bubble.py


def bubble(arr):
    n = len(arr)
    for i in range(n):
        for j in range(0,n-i-1  ):
            if arr[j]>arr[j+1]:
                t=arr[j]
                arr[j]=arr[j+1]
                arr[j+1]=t
    return arr
arr = [3,41,5,2,7,11,63,72,2]
bubble(arr)
print(arr)
    
    